pkgs/development/libraries/glibc/locale-C.patch

```diff
@@ -0,0 +1,3236 @@
+C.utf-8 locales from https://salsa.debian.org/glibc-team/glibc/blob/49767c9f7de4828220b691b29de0baf60d8a54ec/debian/patches/localedata/locale-C.diff
+--- /dev/null
++++ b/localedata/locales/C
```

Comment:
Maybe I can outsource this rather large file.
Since `fetchpatch` is not available while stdenv bootstrapping maybe fetchurl works.

Reply:
`fetchurl` certainly does work.  All sources are fetched that way even during bootstrapping.

Reply:
Yeah. I just switched over.